build.gradle

```diff
@@ -57,6 +74,51 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+task cloneParity(type: com.palominolabs.gradle.task.git.clone.GitCloneTask) {
+  dir = file("$buildDir/parity")
+  uri = 'https://github.com/RightMesh/vagrant-eth-env.git'
+  treeish = 'origin/master' // a commit hash, or tag name, or branch name (with remote prefix, like 'origin/master')
+}
+
+task cloneStatus {
+    doLast {
+        if (cloneParity.state.failure != null) {
+            //here is what shoud be executed if cloneParity fails
+            println 'Failed to use ssh to connect to gitlab. Setup your ssh key and test with ssh -T git@github.com'
+        }
+    }
+}
+
+// creates and starts a parity VM
+task startParity(type: VagrantUp) {
```

Comment:
Very minor.
On ln93 you have
type: VagrantUp
On ln100 you have
type:VagrantDestroy

Spacing should be consistent